govuk-internal/login/resources/css/govuk-internal.css

```diff
@@ -0,0 +1,19 @@
+@charset "UTF-8";
```

Comment:
This looks like a built file. It should be included in the .gitignore and removed from the PR.